### ISSUE TITLE
Version 5.2.4 should be breaking change release

### ISSUE THREAD

**Issue by 0llirocks** [jhund/filterrific#219]
I don't know if you are still using semantic versioning after you changed your versioning strategy to match supported rails version, but version 5.2.4 breaks existing code since you removed jQuery (which is a good thing).

The following line doesn't work anymore and must be changed

Old:
`//= require filterrific/filterrific-jquery`

New:
`//= require filterrific/filterrific`

From my point of view, this is a breaking change and should not be a patch release.

**Comment by MisterGetman** [comment]
Indeed! `//= require filterrific/filterrific-jquery` gave me an error:

> ActionView::Template::Error (couldn't find file 'filterrific/filterrific-jquery' with type 'application/javascript'
